README.md

```diff
@@ -1,6 +1,37 @@
 # Holochain Scaffolding CLI
 
-CLI to easily generate and edit holochain apps.
+A command-line interface for creating and modifying a Holochain application (hApp).
```

Comment:
@pdaoust I'd like to know whether this doc should be in the developer reference rather than here in the scaffolding cli?

Reply:
😬 that is a very good question for which I have no answer. Ideally we'd have documentation 'wherever the dev finds themselves', so I would like to say 'yes to both places'. But that creates duplicated content. I wonder if we could write something that would inline it into the `docs-pages` build pipeline.

Reply:
We can publish this through Rust docs and then add a link to it in the developer guide, what do you think?